### PR TITLE
Wiped and restarted repo

### PR DESCRIPTION
**Pull request by kingram6865**
Issue #7 

Initally created the repo using `expo init` 'blank' option, which made setting up testing more complicated.
When using the 'minimal' option, Jest is included(?). (Still had to install some packages and edit package.json)

Restarted the repo using the 'minimal' configuration which includes Android and iOS base IDE configs.

image

### CODE REVIEW

**Comment by mrmowgli** [comment]
Ok. Assuming that this one includes what's needed for base tests. Could you add back the Routing sample for going back and forth to a different page? After that I will merge.

**Comment by kingram6865** [comment]
> Ok. Assuming that this one includes what's needed for base tests. Could you add back the Routing sample for going back and forth to a different page? After that I will merge.

Yes. Working on putting the routing back in.

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Windows files
```
Comment (mrmowgli) [inline .editorconfig:1]: This file shouldn't be committed. It's specific to your editor. Also the default for git commits should be just line feeds ;)

**Review by mrmowgli — APPROVED** [review]
Looks good to me (LGTM™). Basic Jest snapshots need to be created again, and some of the styling needs to be reapplied from the original sample.Those can be added as a separate issue.

**Pull request merged**